test/uk/gov/hmrc/economiccrimelevyregistration/services/RegistrationValidationServiceSpec.scala

```diff
@@ -42,6 +43,16 @@ class RegistrationValidationServiceSpec extends SpecBase {
 
   val service = new RegistrationValidationService(stubClock, mockSchemaValidator)
 
+  def validateAndCheck(registration: Registration, expected: EclSubscription) = {
+    val result = service.validateRegistration(registration)
+    result shouldBe Valid(Left(expected))
+  }
+
+  def validateAndCheck(registration: Registration) = {
+    val result = service.validateRegistration(registration)
+    result shouldBe Valid(Right(registration))
+  }
```

Comment:
I'm not totally sure I see the benefit of this if I'm honest, it feels like unnecessary abstraction to save a few lines in each test, when the style/pattern in all of our other tests is to just have the call and assertion explicitly within each test so that it's immediately clear when looking at a test exactly what code is being tested and what the assertion is

Reply:
Removed those methods.